workshops/color_scheme_generator/README.md

```diff
@@ -37,14 +37,15 @@ There are many benefits of using Functional components. They are :
 
 ## Setup
 
-For setting up the project fork the repl [here](https://repl.it/@Giridharhackclu/color-scheme-generator-starter#README.md). The starter repl contains ['create-react-app'](https://github.com/facebook/create-react-app) installed. There are some basic styles applied, you will get cleared along with the workshop.
+Fork the starter repl [here](https://repl.it/@Giridharhackclu/color-scheme-generator-starter#README.md). The starter repl contains ['create-react-app'](https://github.com/facebook/create-react-app) installed. There are some basic styles applied, you will get cleared along the flow.
```

Comment:
Phrasing issue is still here: "you will get cleared along the flow"

Reply:
Sorry